app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,79 @@
+      <div class="avatar">
+        <img src="https://www.w3schools.com/howto/img_avatar.png" alt="">
+      </div>
+      <div class="ml-3"><%= post.user.name %></div>
+      <div class="more-actions ml-auto">
+        <i class="fa fa-ellipsis-h fa-large"></i>
+      </div>
+
+    </div>
+    <div class="post-thumbnail">
+      <img src="<%= post.image %>" alt="">
+    </div>
+    <div class="post-action d-flex flex-row align-items-center pl-3 pt-3">
+      <%= link_to post, class: "text-decoration-none" do %>
+        <i class="fa fa-heart-o fa-large text-dark"></i>
+      <% end %>
+      <%= link_to post, class: "text-decoration-none" do %>
+        <i class="ml-3 fa fa-comments fa-large text-dark"></i>
+      <% end %>
+       <%= link_to post, class: "text-decoration-none" do %>
+        <i class="ml-3 fa fa-share fa-large text-dark"></i>
+      <% end %>
+    </div>
+    <div class="post-content pl-3">
+      <div class="post-likes fw-600 mt-1"><%= post.likes.count %> likes</div>
+      <div class="post-body">
+        <span class="fw-600"><%= post.user.name %></span>
+        <span class="font-weight-normal">
+          <%= post.content %>
+        </span>
+      </div>
+    </div>
+    <div class="post-comment pl-3 pr-3">
+      <%= link_to post, class: "text-secondary text-decoration-none" do %>
+        All comments
+      <% end %>
+      <div class="post-body">
+        <span class="fw-600">aaabbbb</span>
+        <span class="font-weight-normal">
+          is simply dummy text of the printing and typesetting industry.
+          is simply dummy text of the printing and typesetting industry.
+          is simply dummy text of the printing and typesetting industry.
+          is simply dummy text of the printing and typesetting industry.
+          is simply dummy text of the printing and typesetting industry.
+          is simply dummy text of the printing and typesetting industry.
+          is simply dummy text of the printing and typesetting industry.
+          is simply dummy text of the printing and typesetting industry.
+          is simply dummy text of the printing and typesetting industry.
+          is simply dummy text of the printing and typesetting industry.
+          is simply dummy text of the printing and typesetting industry.
+          is simply dummy text of the printing and typesetting industry.
+          is simply dummy text of the printing and typesetting industry.
+        </span>
+      </div>
+      <div class="post-body">
+        <span class="fw-600">acccddd</span>
+        <span class="font-weight-normal">font-weight-normal</span>
+      </div>
+      <div class="post-body">
+        <span class="fw-600">eeefff</span>
+        <span class="font-weight-normal">font-weight-normal</span>
+      </div>
+    </div>
+    <div class="post-release-time pl-3">
+      <small class="text-secondary">
+        <%= time_ago_in_words post.created_at %> ago
```

Comment:
It's fine. But you should use the round bracket when pass parameter into a methods
`time_ago_in_words(post.created_at)`
It's easier to understand